corpus/01-component-instr-component/wit/deps/observe/observe.wit

```diff
@@ -0,0 +1,27 @@
+package dylibso:observe;
```

Comment:
It _might_ be worth `.gitignore`-ing this file since we generate it via the `Makefile`?

Reply:
It was needed for cargo test to pass. I reordered building wit / copying it to there so it doesn't need to be in source control.